src/tests/ftest/util/daos_core_base.py

```diff
@@ -126,6 +126,9 @@ def run_subtest(self):
         daos_test_env["COVFILE"] = "/tmp/test.cov"
         daos_test_env["POOL_SCM_SIZE"] = str(scm_size)
         daos_test_env["POOL_NVME_SIZE"] = str(nvme_size)
+        test_log_mask = self.get_test_param("test_log_mask")
+        if test_log_mask is not None:
+            daos_test_env["D_LOG_MASK"] = test_log_mask
```

Comment:
Recommend adjusting line 124 above (where D_LOG_MASK is set) to this, which maintains the same behavior you want
```suggestion
        daos_test_env["D_LOG_MASK"] = self.get_test_param("test_log_mask", "DEBUG")
```

Reply:
+1